content/20.pharmaceuticals.md

```diff
@@ -92,9 +92,10 @@ As results become available from additional clinical trials, we will continue to
 
 ![
 **COVID-19 clinical trials.**
-There are {{ebm_trials}} COVID-19 clinical trials and {{ebm_trials_results}} trials with results as of {{ebm_date_pretty}}.
-The recruitment statuses and trial phases are shown only for trials in which the status or phase is recorded.
-The study types include only types used in at least five trials.
```

Comment:
Should we keep this note about which study types are shown?

Reply:
Yes, I agree we should! Should be fixed in 235e5e3